.github/actions/prepare/action.yml

```diff
@@ -0,0 +1,29 @@
+runs:
+  using: composite
+  steps:
+    - name: Get node version
+      uses: ./.github/actions/get-node-version
+      id: get-node-version
+
+    - name: Get pnpm version
+      id: get-pnpm-version
+      shell: bash
+      run: |
```

Comment:
can we also create an action for this like get-node-version? or inline get-node-version above? (just for consistency)

Reply:
I think it would be better to have a share action that wraps all this but for now it's just a copy from an internal project